src/wifipaf/WiFiPAFEndPoint.cpp

```diff
@@ -797,81 +789,83 @@ SequenceNumber_t WiFiPAFEndPoint::AdjustRemoteReceiveWindow(SequenceNumber_t las
     return static_cast<uint8_t>(newRemoteWindowBoundary - newestUnackedSentSeqNum);
 }
 
-CHIP_ERROR WiFiPAFEndPoint::GetPktSn(Encoding::LittleEndian::Reader & reader, uint8_t * pHead, SequenceNumber_t & seqNum)
+CHIP_ERROR WiFiPAFEndPoint::GetPktSn(Encoding::LittleEndian::Reader reader, SequenceNumber_t & seqNum)
```

Comment:
Passing the reader by value here is weird.  This function then modifies it and whatnot.... but caller will not see those modifications.

I think we should pass it by rvalue reference, to make it clear that the caller is handing over ownership of the reader (at which point the caller could do `GetPktSn(Encoding::LittleEndian::Reader(data->Start(), data->DataLength());` to make it very clear that the reader is not sticking around in the caller).

Reply:
I will change it.